### DISCUSSION TITLE
Clarity wizard left nav buttons not clickable

### DISCUSSION THREAD

**Discussion by datacore-gjanoti** [vmware-clarity/ng-clarity#206] [General]
Hi,

I have been trying to use clarity wizard in one of my Angular project.
But I have been facing issue where the left side nav buttons are not clickable hence not able to navigate to that particular sections through nav buttons.

Please help me finding the issue.
PS- I have followed all the guidelines given in clarity documentation and updated all node modules to latest version

**Comment by kevinbuhmann** [comment]
Can you provide a reproduction on StackBlitz? https://stackblitz.com/@clr-team
